lib/shared/logging/LogUtils.cxx

```diff
@@ -0,0 +1,25 @@
+
+#include <ctime>
+#include <iomanip>
+#include <sstream>
+
+#include <logging/LogUtils.hxx>
+
+std::string
+logging::GetLogName(const std::string& executableName)
+{
+    auto now = time(0);
+    tm* ltm = localtime(&now);
+
+    std::stringstream ss;
+
+    ss << 1900 + ltm->tm_year
```

Comment:
Can you please document the intended format here and perhaps show an example of it in the comment?

Reply:
Added documentation